frontend/www/js/omegaup/problem/mine.ts

```diff
@@ -83,5 +84,52 @@ OmegaUp.on('ready', () => {
       .catch(UI.apiError);
   }
 
+  function visibilityTrue(
```

Comment:
```suggestion
  function normalizeVisibility(
```

normalmente los métodos empiezan con un verbo que indica qué está haciendo.

Reply:
listo